src/com/goide/psi/impl/GoPsiImplUtil.java

```diff
@@ -1441,6 +1474,19 @@ private static GoExpression getLastExpressionAfter(@NotNull List<GoExpression> l
   }
 
   @NotNull
+  private static List<GoExpression> getExpressionsInRange(@NotNull List<GoExpression> list,
```

Comment:
getExpressionsInRange and getExpressionsBefore might be unified in a single method